tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -1557,6 +1557,9 @@ void ExecutionTest::RunLifetimeIntrinsicComputeTest(ID3D12Device *pDevice, LPCST
 
 void ExecutionTest::RunLifetimeIntrinsicLibTest(ID3D12Device5 *pDevice, LPCSTR pShader, CComPtr<ID3D12DescriptorHeap>& pUavHeap, CComPtr<ID3D12RootSignature>& pRootSignature,
                                                 LPCWSTR pTargetProfile, LPCWSTR *pOptions, int numOptions, std::vector<uint32_t> &values) {
+  UNREFERENCED_PARAMETER(values);
+  UNREFERENCED_PARAMETER(pUavHeap);
+
```

Comment:
That fixes the problem, but I don't see any reason to keep these parameters around since they are not used for this or any polymorphic invocation of this function.

Reply:
Probably to keep them the same as the RunLifetimeIntrinsicComputeTest, but you're right - they can be removed.